### DISCUSSION TITLE
Welcome to ixdat Discussions!

### DISCUSSION THREAD

**Discussion by ScottSoren** [ixdat/ixdat#36] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KennethNielsen** [comment]
I'm Kenneth and I work for Spectro Inlets where I write software. Along with this community I hope to be help build an awesome data treatment package :)

**Comment by AlexanderKrabbe** [comment]
Hi all, 

Working with real catalysis at SurfCat DTU, Denmark. 
I'm hoping to use this tool to do analysis on thermal catalysis.

I use a microreactor chip design with slighty different dimensions than the spectro inlets chip [1].

1. I hope to contribute to a definition in the techniques to plot the calibrated mass spectrum signals as a function of recorded temperature from same database entry. 
If one has an idea as to how one would implement this best when the data acquisition does not have the same frequency (len(M28-x) != len(TC-x)), I would be glad to hear. 


2. Along this I also hope to see a technique to calibrate / recalibrate the capillary of each specific chip before/after initiating a new chip. e.i  chip = MSInlet(n_dot=4, carrier='Ar') or chip.recalibrate . Maybe this should can be implemented as aa change in the lentght to an effective length or just add an extra correction parameter in the MSInlet. where n_dot in my case would be calculated from the slope of a pressure increase in the volume on the backside of the capillary before the QMS and in other cases from some known reaction. 


[1] Henriksen, T. R., Olsen, J. L., Vesborg, P. C. K., Chorkendorff, I., & Hansen, O. (2009). Highly sensitive silicon
microreactor for catalyst testing. Review of Scientific Instruments, 80(12), 124101.
https://doi.org/10.1063/1.3270191
